tests/mu_define_enum_test.c

```diff
@@ -18,6 +18,38 @@
 MU_DEFINE_ENUM(TEST4_ENUM, TEST4_ENUM_VALUES);
 MU_DEFINE_ENUM_STRINGS(TEST4_ENUM, TEST4_ENUM_VALUES);
 
+// This lists the values in the enum
+#define TEST5_ENUM_VALUES \
+    test5_a, \
+    test5_b = 42
+
+MU_DEFINE_ENUM(TEST5_ENUM, TEST5_ENUM_VALUES);
+MU_DEFINE_ENUM_STRINGS(TEST5_ENUM, TEST5_ENUM_VALUES);
+
+// This enum has a zero after another non-zero value
+#define TEST6_ENUM_VALUES \
+    test6_a = 42, /* a well known non-zero value */ \
+    test6_b = 0, /* a zero value to check this edge case */ \
+    test6_c = -1, /* a negative value */ \
+    test6_d, /* automagically assigned by the language */ \
+    test6_e, /* another one */ \
+    test6_f = test6_e, /* one that equals the previous value, just to check that the macros can handle it */ \
+    test6_g = test6_f + 1, /* add one with our little hands */ \
+    test6_h = (43 + 42), /* have a paranthesis */ \
+    test6_i = 5 - 1, /* have a minus */ \
+    test6_j = -2 \
```

Comment:
>test6_j = -2 \ [](http://example.com/codeflow?start=4&length=14)

can you add here for giggles test6_k = (1==0)?

Reply:
Added

---
In reply to: [632131277](https://github.com/Azure/macro-utils-c/pull/30#discussion_r632131277) [](ancestors = 632131277)